src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -600,9 +600,9 @@ public function qualifyColumn($column)
      */
     public function qualifyColumns($columns)
     {
-        return (new BaseCollection($columns))->map(function ($column) {
-            return $this->qualifyColumn($column);
-        })->all();
+        return (new BaseCollection($columns))
+            ->map(fn ($column) => $this->qualifyColumn($column))
```

Comment:
This can just be
```suggestion
            ->map([ $this, 'qualifyColumn' ])
```

or
```suggestion
            ->map($this->qualifyColumn(...))
```